tests/ImageSharp.Tests/Processing/Processors/Transforms/EntropyCropTest.cs

```diff
@@ -24,7 +25,16 @@ public class EntropyCropTest
         public void EntropyCrop<TPixel>(TestImageProvider<TPixel> provider, float value)
             where TPixel : unmanaged, IPixel<TPixel>
         {
+            // The result dimensions of EntropyCrop may differ on .NET Core 3.1 because of unstable edge detection results.
+            // TODO: Re-enable this test case if we manage to improve stability.
+#if SUPPORTS_RUNTIME_INTRINSICS
+            if (provider.SourceFileOrDescription.Contains(TestImages.Png.Ducky))
+            {
+                return;
+            }
+#endif
```

Comment:
This is a tricky one, I would probably consider it as a bug, but it should not prevent us from merging this, since jpeg & resize speed is much more important than `EntropyCrop` stability. Probably worth to opening a new issue to track this.

Reply:
Yup... I can have a look at this.